src/components/Form/WysiwygField.js

```diff
@@ -20,48 +19,26 @@ function WysiwygField({
   toolbar = ['INLINESTYLES', 'BLOCKSTYLES', 'OL', 'UL', 'HEADER'],
 }) {
   const {
-    field: { onChange, value, ref },
+    field: { onChange, value },
   } = useController({
     name: nameId,
     control,
-    defaultValue: EditorState.createEmpty(),
+    defaultValue: '<p></p>',
```

Comment:
Is there something in place to prevent an empty `p` tag from being saved to the database?